### ISSUE TITLE
Resume training from a model with different class count. 

### ISSUE THREAD

**Issue by Anesthesic** [open-mmlab/mmdetection#4579]
Hello! I have a dataset with masks with only a subset of them being classified to different labels. 
I would like to start training with all the masks, being agnostic to class types and after that resume training with only the subset that has the class types information. 
Thank you!

**Comment by Shubham-Saket** [comment]
Hello @Anesthesic, 
if the number of classes is different from the pretrained model you can try loading the weights i.e. use load_from instead of resume _from and changing the num_classes in the roi configuration.

**Comment by pushpalatha1405** [comment]
hi usha,

check the file in the path ../od-framework/mmdetection/configs/_base_/models/ cascade_mask_rcnn_r50_fpn.py

In the above file change model settings of the dict key classes to actual number of your custom classes:

ex:

# explicitly over-write all the `num_classes` field from default 80 to 92.
model = dict(
    roi_head=dict(
        bbox_head=[
            dict(
                type='Shared2FCBBoxHead',
                # explicitly over-write all the `num_classes` field from default 80 to 92.
                num_classes=92),
            dict(
                type='Shared2FCBBoxHead',
                # explicitly over-write all the `num_classes` field from default 80 to 92.
                num_classes=92),
            dict(
                type='Shared2FCBBoxHead',
                # explicitly over-write all the `num_classes` field from default 80 to 92.
                num_classes=92)],
    # explicitly over-write all the `num_classes` field from default 80 to 92.
    mask_head=dict(num_classes=92)))

also check the below links for creating and finetuning customized datasets.

https://mmdetection.readthedocs.io/en/latest/tutorials/customize_dataset.html
https://mmdetection.readthedocs.io/en/latest/tutorials/finetune.html
